### PR TITLE
Server routes

### PR DESCRIPTION
**Pull request by adamklingbaum**
- Updates schemas to better reflect data model
- Adds routers/controllers for each endpoint
- Updates API docs to reflect updated schema

To do:
- [ ] Add testing for all endpoints
- [ ] Deploy
- [ ] Provide base URL and Postman instructions to team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,96 +76,62 @@ Returns an event
...
   "colors": {
-    "1": "#F0DA32",
-    "2": "#87B3BF",
-    "3": "#6E4AFF",
-    "4": "#5337C9"
+    "1": "#FFFFFF",
+    "2": "#000000"
   },
+  "__v": 0,
   "guests": {
```
Comment (kyle-sablan) [inline README.md:112]: Would we still want access to each guest's RSVP note? I noticed it was missing in the new readme

**Review by kyle-sablan — APPROVED** [review]
Just need quick clarification on that one comment and then everything should be good to go! Looks great!

File: README.md (hunk 1 above)
Comment (adamklingbaum) [reply README.md:112]: Yes, it is included in the response if there is an `rsvpNote`. These are example guests that had not yet RSVP'd, and if a guest has not yet RSVP'd (rsvpStatus is 'pending'), there will be no `rsvpNote`.

If you go to the example for `GET /events/:eventId/guests` (line 246), you will see it in there.

Comment (adamklingbaum) [reply README.md:112]: I've added another example to the md to make it more clear that guests that have RSVP'd will have their rsvpNote in there"

**Review by hithisiskhai — APPROVED** [review]

**Pull request merged**